faq.html

```diff
@@ -2,7 +2,7 @@
 <html>
 <head>
   <meta charset="utf-8" />
-  <title>AntennaPod | Knowledge Base</title>
+  <title>AntennaPod | F.A.Q.</title>
```

Comment:
@TomHennen How common is the usage of this abbreviation in the US? Would, say, grandparents understand?

Also, I'm not sure that FAQ is really better than knowledge base. While most items here are questions, I'm not sure that it will stay this way. This is also meant as an tutorial to get people started.